docs/source/about/index.rst

```diff
@@ -0,0 +1,75 @@
+Decoupling these two steps yields a powerful and flexible modeling framework as many analysis steps are independent
+of the construction of the transition matrix. For example, whether we use RNA velocity or a pseudotime to derive
+directed transition probabilities does not change how initial and terminal states are inferred or fate probabilities
+estimated. The general structure of the framework, corresponding to steps **(i)** and **(ii)**, is given by:
+
+* :ref:`Kernels <kernels>` that take multi-view single cell input data  and estimate a matrix of cell-cell transition
+  probabilities :math:`T`. Row :math:`i` in matrix :math:`T` contains the transition probabilities from cell :math:`i`
+  towards putative descendants. Therefore, all entries in the matrix are between 0 and 1, and rows sum to one.
+* :ref:`Estimators <estimators>` that take a cell-cell transition matrix :math:`T` computed using any kernel and apply
+  concepts from the theory of Markov chains to identify initial, terminal, and intermediate
+  :doc:`macrostates <../notebooks/tutorials/initial_terminal_states>` and compute
+  :doc:`fate probabilities <../notebooks/tutorials/fate_probabilities>`.
+
+Our main (and recommended!) estimator is based on *Generalized Perron Cluster Cluster Analysis* (GPCCA)
+:cite:`reuter:18,reuter:19`, a method originally developed to study conformational protein dynamics. CellRank uses a
+robust implementation of GPCCA through the `pyGPCCA`_ package.
+Please don't forget to cite both CellRank and GPCCA when using the :class:`cellrank.estimators.GPCCA` estimator,
+see :doc:`How to cite CellRank <about/cite>`.
+
+We use fate probabilities to visualize trajectory-specific
+:doc:`gene expression trends <../notebooks/tutorials/gene_trends>`, infer putative
+:doc:`driver genes <../notebooks/tutorials/fate_probabilities>`, arrange cells in a
+:func:`circular embedding <cellrank.pl.circular_projection>` :cite:`velten:17`, visualize
+:func:`cascades of gene activation <cellrank.pl.heatmap>` along a trajectory, and
+:func:`cluster expression trends <cellrank.pl.cluster_trends>`.
+
+.. _sparsity:
+
+Sparsity
+~~~~~~~~
+All CellRank kernels yield sparse transition matrices :math:`T`. Further, the :class:`cellrank.estimators.GPCCA`
+estimator exploits sparsity in all major computations. Sparsity allows CellRank to scale to large datasets.
+
+Why is it called "CellRank"?
+----------------------------
+CellRank **does not** rank cells, we gave the package this name because just like Google's original `PageRank`_
+algorithm, it works with Markov chains to aggregate relationships between individual objects (cells vs. websites)
+to learn about more global properties of the underlying dynamics (initial & terminal states and fate probabilities vs.
+website relevance).
+
+.. _PageRank: https://en.wikipedia.org/wiki/PageRank
+.. _pyGPCCA: https://pygpcca.readthedocs.io/
```

Comment:
Not sure, if we should link the RTD or GitHub page (https://github.com/msmdev/pyGPCCA) here.

Reply:
your call - feel free to change.